python/py_vapid/main.py

```diff
@@ -25,7 +25,7 @@ def main():
     parser.add_argument('--sign', '-s', help='claims file to sign')
     parser.add_argument('--gen', '-g', help='generate new key pairs',
                         default=False, action="store_true")
-    parser.add_argument('--version2', '-2', help="use VAPID spec Draft-02",
+    parser.add_argument('--version2', '-2', help="use RFC8292 VAPID spec",
```

Comment:
maybe make this the default? default is `False` here, but `True` below.

Reply:
ah, good catch, thanks!